tests/tests_pytorch/strategies/test_ddp.py

```diff
@@ -56,22 +56,6 @@ def test_multi_gpu_model_ddp_fit_test(tmpdir):
         assert out["test_acc"] > 0.7
 
 
-@RunIf(skip_windows=True)
-def test_torch_distributed_backend_invalid(cuda_count_2, tmpdir):
-    """This test set `undefined` as torch backend and should raise an `Backend.UNDEFINED` ValueError."""
-    model = BoringModel()
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        fast_dev_run=True,
-        strategy=DDPStrategy(process_group_backend="undefined"),
-        accelerator="cuda",
-        devices=2,
-        logger=False,
-    )
-    with pytest.raises(ValueError, match="Invalid backend: 'undefined'"):
```

Comment:
PyTorch 2.0 changed the backend validation logic, and this is now hanging instead of raising the ValueError

Either way, we can remove this test because the exception is raised inside Torch, so we don't need to test that